lan867x/README.md

```diff
@@ -26,23 +26,34 @@ esp_eth_phy_t *phy = esp_eth_phy_new_lan867x(&phy_config);
 
 #### You may need to setup the PLCA
```

Comment:
This section could have a better title.

Reply:
Will be addressed in the next commit along with other documentation changes.